invenio_cli/commands/upgrade.py

```diff
@@ -52,3 +52,32 @@ def upgrade():
         ]
 
         return steps
+
+    @staticmethod
+    def update_app():
+        """Upgrades the version of invenio-app-rdm 3.0.0.
+
+        It is a class method since it does not require any configuration.
+        """
+        prefix = ['pipenv']
+        app = prefix + [
+            'install', 'invenio-app-rdm[postgresql, elasticsearch7]~=3.0.0'
```

Comment:
Just to account for possible minor updates:

```suggestion
            'install', 'invenio-app-rdm[postgresql, elasticsearch7]~=3.0'
```
(i.e. this allows update to 3.1.0 if ever we do release that, while not upgrading to 4.0 if it exists)

Overall it does tie the invenio-cli code with the instance version i.e. 
- for invenio-cli developers, we need to update this piece of code for each release
- for administrators, upgrading invenio-cli would cause this command to upgrade their instance perhaps unwillingly

In the future we could parse the Pipfile and use the version constraints for invenio-app-rdm found there to circumvent that problem. Make an issue for this if not tackled now.